packages/wifi_iot/ios/Classes/SwiftWifiIotPlugin.swift

```diff
@@ -106,18 +106,15 @@ public class SwiftWifiIotPlugin: NSObject, FlutterPlugin {
     private func forceWifiUsage(call: FlutterMethodCall, result: FlutterResult) {
         let arguments = call.arguments
         let useWifi = (arguments as! [String : Bool])["useWifi"]
-        if (useWifi != nil) {
-            print("Forcing WiFi usage : %s", ((useWifi ?? false) ? "Use WiFi" : "Use 3G/4G Data"))
-            if #available(iOS 14.0, *) {
-                if(useWifi ?? false){
-                    // trigger access for local network
-                    triggerLocalNetworkPrivacyAlert();
-                }
-            } else {
-                result(FlutterMethodNotImplemented)
+        print("Forcing WiFi usage : %s", ((useWifi ?? false) ? "Use WiFi" : "Use 3G/4G Data"))
+        if #available(iOS 14.0, *) {
+            if(useWifi ?? false){
+                // trigger access for local network
+                triggerLocalNetworkPrivacyAlert();
             }
+            result(true)
         } else {
-            result(nil)
+            result(FlutterMethodNotImplemented)
```

Comment:
Looking at the Dart code for this call, it only ever returns `true` or `null`. Instead of returning `FlutterMethodNotImplemented`, it might make sense to return `false` for the sake of consistency.

Furthermore, the Dart method should safely get the result without having to return a nullable `bool`:

```dart
bool result = false;
try {
  result = await _channel.invokeMethod('forceWifiUsage', htArguments) ?? false; 
} on MissingPluginException catch (e) {
  print("MissingPluginException : ${e.toString()}");
}
return result;
```

Reply:
Done necessary changes with https://github.com/alternadom/WiFiFlutter/pull/213/commits/467df493c3da29bb98b56b1cf6db624a707d652d